nuxeo-core/nuxeo-core-io/src/main/java/org/nuxeo/ecm/core/io/marshallers/json/document/DocumentModelJsonReader.java

```diff
@@ -122,7 +122,9 @@ protected DocumentModel readEntity(JsonNode jn) throws IOException {
                 if (!propertyName.contains(":")) {
                     propertyName = property.getField().getDeclaringType().getName() + ":" + propertyName;
                 }
-                simpleDoc.setPropertyValue(propertyName, property.getValue());
+                String pn = propertyName;
+                // security will apply in previous step when reading properties
```

Comment:
In 10.10 we had changed it to // security has been applied in previous step while reading properties (clearer)